pkg/kopia/command/storage/filesystem_args.go

```diff
@@ -0,0 +1,21 @@
+package storage
+
+import (
+	v1 "k8s.io/api/core/v1"
+
+	"github.com/kanisterio/kanister/pkg/logsafe"
+)
+
+const (
+	filesystemSubCommand = "filesystem"
+	pathFlag             = "--path"
+	// DefaultFSMountPath is the mount path for the file store PVC on Kopia API server
+	DefaultFSMountPath = "/mnt/data"
```

Comment:
How about we keep the mount path as `/mnt/secrets/storage/fs`? I have been keeping `/mnt/secrets/` as the root path of all the secret mount paths cc: @pavannd1